docs/developer-docs/setup/install/index.mdx

```diff
@@ -105,16 +105,30 @@ generateResolvConf = false`}
 
 ## What gets installed
 
-The SDK installation script installs several components in default locations on your local computer. The following table describes the development environment components that the installation script installs:
+The IC SDK installation script installs several components in default locations on your local computer. The following table describes the development environment components that the installation script installs:
 
 | Component    | Description                                                                                        | Default location                              |
 |--------------|----------------------------------------------------------------------------------------------------|-----------------------------------------------|
-| dfx          | DFINITY execution command-line interface (CLI)                                                     | `/usr/local/bin/dfx`                          |
+| dfx          | Command-line interface (CLI)                                                     | `/usr/local/bin/dfx`                          |
+| cdk-rs       | Rust CDK                                                       | `/usr/local/bin/dfx`                          |
```

Comment:
cdk-rs isn't actually installed by, or contained within, dfx; this one should probably be reverted

Reply:
removed